release.nix

```diff
@@ -1,15 +1,31 @@
 let
-  overlay = self: super:
-{
-  random-fu     = self.haskellPackages.callPackage ./random-fu { rvar = self.rvar; };
-  random-source = self.haskellPackages.callPackage ./random-source { };
-  rvar          = self.haskellPackages.callPackage ./rvar { random-source = self.random-source; };
+  myHaskellPackageOverlay = self: super: {
+    myHaskellPackages = super.haskellPackages.override {
+    overrides = hself: hsuper: rec {
+        random = hsuper.random_1_2_0;
+        mkDerivation = args: hsuper.mkDerivation (args // {
+          doCheck = false;
+          doHaddock = false;
+        });
+
+    random-fu     = super.haskell.lib.disableLibraryProfiling (hself.callPackage  ./random-fu { });
+    random-source = super.haskell.lib.disableLibraryProfiling (hself.callPackage ./random-source { random = hsuper.random_1_2_0; });
+    rvar          = super.haskell.lib.disableLibraryProfiling (hself.callPackage ./rvar { });
+    splitmix      = hsuper.splitmix_0_1;
+    MonadRandom   = hsuper.MonadRandom_0_5_2;
+    };
+  };
```

Comment:
Not sure what's going on here. Does this pin `random` to `1.2.0`? And what goes wrong with profiling?

Reply:
I removed it